helpers/spark-defaults.conf

```diff
@@ -0,0 +1,8 @@
+spark.app.name=spark-app
+spark.executor.instances=2
+spark.kubernetes.container.image=docker.io/averma32/sparkpy6:latest
+spark.kubernetes.container.image.pullPolicy=IfNotPresent
+spark.kubernetes.namespace=default
+spark.kubernetes.authenticate.driver.serviceAccountName=spark
+spark.eventLog.enabled=false
+spark.driver.extraJavaOptions='-Dscala.shell.histfile=${SNAP_USER_COMMON}/.scala_history'
```

Comment:
I have tested this but in my environment $SNAP_USER_COMMON is not defined. Beside, if I substitute $HOME and provide access with "snap connect spark-clinent:dot-scala-history", the auto-completion in the shell still does not seem to work. I'm afraid env variables in the spark-conf are not resolved. 

We should probably implement this via the configuration management proposed in the doc. Actually this flag could potentially be applied dynamically only for spark-shell commands, as this configuration should not really matter for spark-submit jobs, should it? 

Anyhow, as far as this PR is concerned, we could probably skip this configuration for now and delete this line